flow-sample-basic/src/main/java/flow/sample/basic/HelloScreen.java

```diff
@@ -0,0 +1,47 @@
+package flow.sample.basic;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+final class HelloScreen implements Parcelable {
```

Comment:
Sounds distracting.  I don't know that we can save people from knowing about Parcelable, or that I want to try.